test/Market.t.sol

```diff
@@ -97,7 +97,7 @@ contract Market_swap_revertsWhenUnsafeERC20TransferFails is UpgradeableMarket {
   }
 
   function test() external {
-    vm.startPrank(owner);
+    vm.prank(owner);
```

Comment:
switched to using `vm.prank` in several places to minimize the number of places that the msg.sender deviates from the expected global default value.